### PR TITLE
fix filter for measures on page

### PR DESCRIPTION
**Pull request by riedde**
Before contains() was used but this produces too many matches if the IDs contains floating numbers (multiple matches). Now the exact id must occur for a match.

### CODE REVIEW

**Comment by riedde** [comment]
closes #218

**Comment by bwbohl** [comment]
I will test this

File: add/data/xql/getMeasuresOnPage.xql [hunk 1]
```diff
@@ -44,7 +44,7 @@ declare option exist:serialize "method=text media-type=text/plain omit-xml-decla
 declare function local:getMeasures($mei as node(), $surface as node()) as xs:string* {
 
     for $zone in $surface/mei:zone[@type='measure']
-    let $measures := $mei//mei:measure[contains(@facs, concat('#', $zone/@xml:id))]
+    let $measures := $mei//mei:measure[@facs = concat('#', $zone/@xml:id)]
```
Comment (bwbohl) [inline add/data/xql/getMeasuresOnPage.xql:47]: please consider the case that a `@facs` can contain multiple references, e.g. by tokenizing the attribute value and testing against the set of tokens

**Review by bwbohl — CHANGES_REQUESTED** [review]

File: add/data/xql/getMeasuresOnPage.xql (hunk 1 above)
Comment (riedde) [reply add/data/xql/getMeasuresOnPage.xql]: done

**Pull request merged**